src/components/Band.js

```diff
@@ -1,19 +1,14 @@
-import React, { Component } from 'react';
+import React from 'react';
 
-class Band extends Component {
+const Band = ({ deleteBand, id, name }) => {
```

Comment:
So, this is just object destructuring. We know `deleteBrand` et al are coming in as props. By leaning on that ES6 we can skip having to reference `this.props.WHATEVER` throughout the code. An alternative for when you _need_ to use a class component is to destructure off of `this.prop` like so:

```
class Band extends Component {
  const { deleteBand } = this.props;

  render()  {
    ...
  }
}
```

Reply:
You'll also note that I've converted away from a class-style component in favor of a ho-hum functional component. As a general rule, if you do not need the component life-cycle hooks (`componentDidMount`, etc) there's no need to extend the heavier `React.Component` and include them. This will pay dividends in larger-scale production environments.